spec/helpers/admin/claims_helper_spec.rb

```diff
@@ -71,11 +78,20 @@
   end
 
   describe "admin_student_loan_details" do
+    let(:eligibility) { build(:student_loans_eligibility, employments: [employment]) }
+
+    let(:employment) do
+      build(
+        :student_loans_employment,
+        student_loan_repayment_amount: 1234,
+      )
+    end
+
     let(:claim) do
       build(
         :claim,
         student_loan_plan: :plan_1,
-        eligibility: build(:student_loans_eligibility, student_loan_repayment_amount: 1234),
+        eligibility: eligibility,
       )
     end
```

Comment:
This can be further simplified by inlining the `employment` and `eligibility` builds in `claim`.